lib/tasks/full_import.rake

```diff
@@ -0,0 +1,405 @@
+    # Remove trailing empty columns
+
+  #<CSV::Row
+    # date:"2/2/2025"
+    # last_f_contact:"7/1/2025"
+    # last_pm_update:nil
+    # private:"from Jen"
+    # nil:"C"
+    # status:"REMATCH"
+    # notes:"7/1: asked for rematch, waiting for photos. 4/28: working on it"
+    # terms:"Y"
+    # brands:nil
+    # project_name:"Crochet Lap blanket In Almonte, ON"
+    # finisher_name:"redacted - potential, email drafted\nF1 redacted"
+    # finisher_address:"164 redacted st"
+    # finisher_city:"Redacted"
+    # finisher_state:"ON"
+    # finisher_country:"K0A1A0"
+    # finisher_zip:"CA"
+    # finisher_email:"redacted@gmail.com"
+    # finisher_phone:"613-555-1212"
+    # project_owners_name:"Redacted"
+    # project_owner_address:"102 Redacted Street North"
+    # city:"Almonte" state:"ON" zip:"K0a1a0"
+    # project_owner_email:"redacted2@gmail.com"
+    # project_owner_phone:"613-555-1212"
+    # original_crafter:"Arlette Redacted"
+    # who_for:"For herself I believe "
+    # project_description:"Lap blanket"
+    # pattern:"No"
+    # project_photo:"https://drive.google.com/open?id=1CtdRdTZQOqivhoyIHJAQe0Re8XFEMFp0"
+    # materials_photo:"https://drive.google.com/open?id=19DNZBiSItggrSbcayNnPUX0FWH3PRpX_"
+    # project_info:nil
+    # pattern_photo:nil
+    # crafter_bio:"My mother Arlette has always been creative, and loved textiles. As a young newlywed she starts this lap blanket. Life and babies got in the way and it was stored away, but survived several house moves. Mum later became an avid and talented knitter, but in 2023 she started to become dissatisfied with her knitting, pulling a finished project apart if it had the most inconsequential flaw. Dad could not understand why. Then in early 2024 she was diagnosed with Lewy Body dementia and declined very quickly. She is no longer able to focus or use her hands and is now in a care home. It would be lovely if this project could be finished for her. Unfortunately my siblings and I are not knitters or crocheters. It would be my hope that the finished project might stir some memories and be a comfort to her. \nMany thanks for your consideration and for the special work that you do."
+    # crafter_picture:"https://drive.google.com/open?id=1-ZUj0fW009MG3YiY79BBNDwfEaEx9XSi"
+    # craft:"Crochet"
+    # material:"Synthetic (Nylon, Polyester, Acrylic, etc)"
+    # referred:"Instagram"
+    # terms:"I agree."
+    # project_pic_ok_to_use:"Yes"
+    # crafter_photo_ok_to_use:"No"
+    # pets_ok:"Yes"
+    # po_pets:nil
+    # po_smoker:nil
+    # right_left:nil
+    # finisher_feedback_form_sent:nil
+    # po_feedback_form_sent:nil
+    # po_feedback_form_recd:nil
+  #>
+
+  CSV_KEY = "latest_sheet_dump.csv"
+  BUCKET = "looseendsproject-" + ENV["RAILS_ENV_DISPLAY"]
+  TMPFILE = Rails.root.join("tmp", "full_import.csv")
+  DEFAULT_PASSWORD = "CREATED_FROM_IMPORT"
+
+  desc "Import Google Sheet data"
+  task import: [:environment] do |_t|
+    MANAGER_JEAN = User.find(108513)
```

Comment:
Might be easier to understand as lookup by email, but that's really the tiniest of nits